Deployment environment variable group creation as part of provisioning. (#184)
devonfw/hangar: documentation/azure-devops/setup-eks-provisioning-pipeline.asciidoc

=== Flags
```
-c, --config-file               [Required] Configuration file containing pipeline definition.
-n, --pipeline-name             [Required] Name that will be set to the pipeline.
-d, --local-directory           [Required] Local directory of your project (the path should always be using '/' and not '\').
    --cluster-name              [Required] Name for the cluster."
    --s3-bucket                 [Required] Name of the S3 bucket where the Terraform state of the cluster will be stored.
    --s3-key-path               [Required] Path within the S3 bucket where the Terraform state of the cluster will be stored.
    --aws-access-key            [Required, on first run] AWS account access key ID. 
    --aws-secret-access-key     [Required, on first run] AWS account secret access key.

>     --aws-region                [Required, on first run] AWS Region to provision the resources.

Reviewer: ```suggestion
    --aws-region                [Required, on first run] AWS region for provisioning resources.
```
